Source/WebGPU/WebGPU/Device.mm

```diff
@@ -136,7 +136,7 @@ static void captureFrame(id<MTLDevice> captureObject)
 
 Ref<Device> Device::create(id<MTLDevice> device, String&& deviceLabel, HardwareCapabilities&& capabilities, Adapter& adapter)
 {
-    id<MTLCommandQueue> commandQueue = [device newCommandQueue];
+    id<MTLCommandQueue> commandQueue = [device newCommandQueueWithMaxCommandBufferCount:2048];
```

Comment:
Benchmark showed there was no real harm in raising this number, only when a large number of command buffers are used is there degradation. I tried all the way to UINT_MAX and I didn't observe a difference from the default value of 64

Reply:
Why 2048 when the max available to the API is 1000?

Reply:
We should be able to make this 1000 but my concern is some JavaScript object or other object accidentally retaining a command buffer that has not been submitted and otherwise has zero references to it.

Then we have a MTLQueue with the max count and `commandBufferWithDescriptor:` will hang until commit is called on a MTLCommandBuffer which might never occur (so the WebGPU thread hangs indefinitely in the GPU process).

We really don't want to hang the thread which calls `commandBufferWithDescriptor:` when we reach this limit. I hesitate to use UINT_MAX here only because I did not test various Intel and AMD drivers with that value. Passing UINT_MAX on Apple Silicon and iOS seemed to not degrade performance as long as a small number of command buffers was actually used.